extensions/wikia/ImageReview/ImageCountGetter.class.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class ImageCountGetter extends WikiaModel {
+
+	const QUESTIONABLE = 'questionable';
+	const REJECTED     = 'rejected';
+	const UNREVIEWED   = 'unreviewed';
+	const INVALID      = 'invalid';
+
+
+	/**
+	 * Fix this so it returns a count of all available images to see AND all
+	 * images already assigned to this user, but that they haven't taken an
+	 * action on
+	 * @return array
+	 * @throws DBUnexpectedError
+	 * @throws MWException
+	 */
+	public function getImageCounts( $reviewerId ) {
+		$counts = $this->mergeCounts(
+			$this->getUnassignedImageCounts(),
+			$this->getUsersInProgressImageCounts( $reviewerId )
+		);
+
+		return [
+			self::QUESTIONABLE => $counts[ ImageReviewStates::QUESTIONABLE ] ?? 0,
+			self::REJECTED => $counts[ ImageReviewStates::REJECTED ] ?? 0,
+			self::UNREVIEWED => $counts[ ImageReviewStates::UNREVIEWED ] ?? 0,
+			self::INVALID => $counts[ ImageReviewStates::INVALID_IMAGE ] ?? 0
+		];
+	}
+
+	private function getUnassignedImageCounts() {
+		$db = $this->getDatawareDB();
+		$counts = ( new WikiaSQL() )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$counts". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)